src/systems/GameSystem.cpp

```diff
@@ -93,6 +369,136 @@ namespace indie
         }
     }
 
+    //     std::shared_ptr<Entity> e3 = std::make_shared<Entity>();
+    //     std::shared_ptr<Position> pos3 = std::make_shared<Position>(10, 0, 0);
+    //     std::shared_ptr<String> text = std::make_shared<String>("The below sprite entity has a hitbox of 250,250");
+
+    //     std::shared_ptr<Entity> e4 = std::make_shared<Entity>();
+    //     std::shared_ptr<Position> pos4 = std::make_shared<Position>(0, 0, 0);
+    //     std::shared_ptr<Grid> grid = std::make_shared<Grid>(10, 1.0f);
+
+    //     std::shared_ptr<Entity> e = std::make_shared<Entity>();
+    //     std::shared_ptr<Rect> rect  = std::make_shared<Rect>(0, 0, 0, 0);
+    //     std::shared_ptr<Position> pos = std::make_shared<Position>(500, 500);
+    //     std::shared_ptr<Sprite> sprite = std::make_shared<Sprite>("test_pictures/scarfy.png", 6);
+
+    //     std::shared_ptr<Entity> e2 = std::make_shared<Entity>();
+    //     std::shared_ptr<Position> pos2 = std::make_shared<Position>(0, 0, 0);
+    //     std::shared_ptr<Model3D> model = std::make_shared<Model3D>("test_models/turret.obj", "test_models/turret_diffuse.png");
+
+    //     std::shared_ptr<Entity> entity2 = std::make_shared<Entity>();
+    //     std::shared_ptr<Position> component = std::make_shared<Position>(10, 10);
+    //     std::shared_ptr<Sprite> component4 = std::make_shared<Sprite>("test_pictures/raylib_logo.png");
+    //     std::shared_ptr<Rect> component5 = std::make_shared<Rect>(0, 0, 250, 250);
+
+
+    //     scene->addEntities({entity2, e, e4, e2, e3});
+    //     scene->addEntities({createCamera(camPos,camTarget)});
+    //     generateMap("assets/maps/map2.txt", *scene);
```

Comment:
same thing why so much things in comment ?

Reply:
Fix in last commits